shell.nix

```diff
@@ -9,8 +9,13 @@
 }:
 with pkgs;
 let
+  ghcVersion = "ghc8104";
+
   # This is temporally till we get a stylish 0.12.3.0 release available in hackage
-  stylish-haskell = import ./nix/stylish-haskell.nix { inherit pkgs; };
+  stylish-haskell = import ./nix/stylish-haskell.nix { inherit pkgs ghcVersion; };
+
+  hls = import ./nix/hls.nix { inherit pkgs ghcVersion; };
```

Comment:
As with `stylish-haskell` above, this is also temporary until eg `1.1.0.0` is on Hackage, right?

For both of these, once it's on Hackage, we can juse use the `tools` field, right? (And simultaneously remove them from the `sources.json`.)

Might be helpful to update these comments to say as much.

Reply:
Reworded the comment, adding some explanation about why this is here, and what to do when the relevant versions arrive to Hackage. :+1: